src/utils/pdf.py

```diff
@@ -324,10 +349,10 @@ def render_rules():
         "En cas d'égalité, le dernier joueur éliminé aura le dernier mot. Les modalités de vote",
         'Le dernier vote (lors de la finale) est ouvert à tous les éliminés et dure du 8h00 à 20h59. A partir de l’ouverture de ce vote, les finalistes se voient retirer leur droit de parole sur l’ensemble des salons.',
         'Une fois éliminé, les personnes ne peuvent plus participer aux votes, ne peuvent plus interagir sur le groupe de discussion général, peuvent interagir sur le groupe des éliminés.',
-        'Un système de collier d’immunité a été mis en place. En voici les règles :',
+        'Un système de collier d’immunité a été mis en place. En voici les règles:',
         [
-            'Un smiley :collierimmunite:  est dissimulé quelque part sur le serveur, dans une discussion accessible à tous les joueurs restants, à une date tenue secrète.',
-            'Toute personne qui trouve le smiley :collierimmunite:  est enjoint à cliquer dessus au plus vite. Ce dernier disparaitra de façon quasi instantanée. Dès à présent, vous êtes détenteur (de façon secrète) du collier d’immunité.',
+            'Un smiley:collierimmunite:  est dissimulé quelque part sur le serveur, dans une discussion accessible à tous les joueurs restants, à une date tenue secrète.',
```

Comment:
Il manque un espace après le mot "smiley"
```suggestion
            'Un smiley :collierimmunite:  est dissimulé quelque part sur le serveur, dans une discussion accessible à tous les joueurs restants, à une date tenue secrète.',
```